data-ingestion-service/src/main/java/gov/cdc/dataingestion/nbs/ecr/service/helper/CdaXmlAnswerMappingHelper.java

```diff
@@ -50,8 +50,10 @@ private POCDMT000040Component3 mapToExtendedData(EcrMsgXmlAnswerDto in, POCDMT00
         try {
             String xmlContent = in.getAnswerXmlTxt();
             if (xmlContent != null && !xmlContent.isEmpty()) {
+                xmlContent = xmlContent.replaceFirst("sdt:dischargeDispositionCode",
+                        "sdt:dischargeDispositionCode xmlns:sdt=\"urn:hl7-org:sdt\"");
                 DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-                factory.setNamespaceAware(false);
+                factory.setNamespaceAware(true);
```

Comment:
have to set this to true so the xml convert wont remove sdt namespace, but the input payload missing xmlns:sdt=\"urn:hl7-org:sdt\" hence it without the modification on line 53 while the setNamespaceAware is true the code will throw some kind of std unbound exception